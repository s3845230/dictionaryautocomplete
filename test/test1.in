TS
A booming 123456
TE
TS
A sysop  526687
TE
TS
A droege  49660
TE
TS
A mumbled  276990
TE
TS
A opensubscriber  58547
TE
TS
A underappreciated  151730
TE
TS
A sampai  35185
TE
TS
A trees  27378116
TE
TS
A nickelodeon  813928
TE
TS
A quintuple  74194
TE
TA