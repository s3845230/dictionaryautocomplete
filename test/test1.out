Found 'btvinfo' with frequency 176464
Found 'potentiate' with frequency 67781
Found 'skywalker' with frequency 656667
Found 'uhu' with frequency 106193
Found 'mbw' with frequency 58947
Found 'medicina' with frequency 498885
Found 'pickstop' with frequency 68019
Found 'crackhead' with frequency 85151
Found 'lazear' with frequency 62101
Found 'spoleto' with frequency 141214
Found 'rdma' with frequency 88767
Found 'decommissioning' with frequency 744603
Found 'wrexham' with frequency 896167
Found 'helfrich' with frequency 56214
Found 'variegated' with frequency 386434
Found 'hitch' with frequency 2730000
Found 'odsp' with frequency 35936
Found 'devoir' with frequency 75174
Found 'counterclockwise' with frequency 266544
Found 'coull' with frequency 32060
Found 'profligacy' with frequency 37329
Found 'loic' with frequency 173130
Found 'wookie' with frequency 124578
Found 'saale' with frequency 80902
Found 'indirection' with frequency 143143
Found 'popmatters' with frequency 185930
Found 'aukcje' with frequency 55391
Found 'unifrance' with frequency 31584
Found 'gameface' with frequency 34942
Found 'failsworth' with frequency 34284
Found 'alphecca' with frequency 104572
Found 'clubman' with frequency 116626
Found 'chapstick' with frequency 62478
Found 'boondoggle' with frequency 80270
Found 'altough' with frequency 62984
Found 'cortex' with frequency 2530902
Found 'charlier' with frequency 40078
Found 'cutler' with frequency 1231345
Found 'lanway' with frequency 31929
Found 'mullica' with frequency 73343
Found 'omr' with frequency 439577
Found 'emulating' with frequency 297217
Found 'isoprene' with frequency 85972
Found 'rojas' with frequency 438182
Found 'catwalk' with frequency 572806
Found 'steeper' with frequency 350882
Found 'broeck' with frequency 39549
Found 'neckwear' with frequency 163236
Found 'wwwanywho' with frequency 31001
Autocomplete for 'dav': [ daventry: 272116  davidso: 127910  davidm: 41731  ]
Autocomplete for 'tst': [ tstyle: 30935  ]
Autocomplete for 'som': [ someplace: 598811  somaliland: 347056  sombre: 246150  ]
Autocomplete for 'ase': [ ases: 274362  asea: 74230  asesu: 70868  ]
Autocomplete for 'gri': [ grill: 10254124  grimes: 1010676  grilles: 672094  ]
Autocomplete for 'oza': [ ozarks: 485034  ]
Autocomplete for 'gfd': [ gfd: 73034  ]
Autocomplete for 'sea': [ sealing: 2380108  seaworld: 578580  seamed: 334904  ]
Autocomplete for 'ttr': [ ttr: 172521  ]
Autocomplete for 'cre': [ create: 137071122  creator: 11071006  creations: 5273628  ]
Autocomplete for 'gru': [ grudgingly: 160743  grungy: 128792  gruss: 97365  ]
Autocomplete for 'fub': [ fub: 33805  ]
Autocomplete for 'ces': [ cessation: 2137188  cespedes: 34649  ]
Autocomplete for 'str': [ strengthening: 5079049  stretches: 1725140  stratton: 1272445  ]
Autocomplete for 'pow': [ powder: 12627415  poway: 411098  powerplay: 302761  ]
Autocomplete for 'fen': [ fen: 1028033  fencer: 119126  fense: 55120  ]
Autocomplete for 'mad': [ maddison: 132248  madhavan: 86395  madrazo: 42600  ]
Autocomplete for 'dhc': [ dhcpd: 134837  ]
Autocomplete for 'far': [ farnborough: 444459  farberware: 323766  farrer: 138832  ]
Autocomplete for 'sob': [ soberly: 75811  sobered: 58791  sobase: 40348  ]
Autocomplete for 'con': [ conditions: 168957006  contents: 78365831  contacts: 34965494  ]
Autocomplete for 'dub': [ dubbed: 1749520  dubious: 1252536  dubble: 66517  ]
Autocomplete for 'aub': [ aubry: 113051  ]
Autocomplete for 'non': [ nondurable: 291028  nona: 224266  nonemployers: 139668  ]
Autocomplete for 'rea': [ reasonably: 8548385  realist: 623441  ream: 470230  ]
Autocomplete for 'adv': [ advance: 44738913  advise: 8325087  advertisements: 7341945  ]
Autocomplete for 'wie': [ wie: 2167727  wiesel: 242306  wiese: 174956  ]
Autocomplete for 'nal': [ nalc: 45275  nales: 30921  ]
Autocomplete for 'int': [ integrated: 37600088  interference: 6209872  interpreter: 3858537  ]
Autocomplete for 'caz': [ cazuza: 287301  cazzo: 211467  ]
Autocomplete for 'gua': [ guardians: 2007578  guarding: 1005862  guardianfilms: 413380  ]
Autocomplete for 'sch': [ schreiber: 649483  scholes: 436993  schip: 314197  ]
Autocomplete for 'kla': [ klang: 309031  klar: 129047  ]
Autocomplete for 'ecp': [ ecp: 515274  ]
Autocomplete for 'con': [ conditions: 168957006  contents: 78365831  contacts: 34965494  ]
Autocomplete for 'clu': [ clutched: 202319  cluck: 134048  cluttering: 129476  ]
Autocomplete for 'res': [ research: 311538382  resource: 99964083  restaurant: 48255033  ]
Autocomplete for 'geb': [ gebundene: 51912  gebhart: 45585  gebauer: 36747  ]
Autocomplete for 'pen': [ pentax: 4285966  penetrating: 1327533  penrith: 733803  ]
Autocomplete for 'fun': [ functional: 41862937  funeral: 12827388  functioning: 5619525  ]
Autocomplete for 'may': [ maynard: 1548244  maywood: 415662  mayday: 311257  ]
Autocomplete for 'sup': [ supplement: 12988231  supra: 4329457  superoxide: 469300  ]
Autocomplete for 'epx': [ epx: 252273  ]
Autocomplete for 'lab': [ laboratory: 31956885  labeled: 5765620  labourer: 952092  ]
Autocomplete for 'myn': [ mynx: 97271  ]
Autocomplete for 'can': [ cancelled: 5573318  canonical: 1982599  canaria: 902712  ]
Autocomplete for 'sca': [ scary: 5571680  scarce: 2385219  scarcity: 902492  ]
Autocomplete for 'dev': [ device: 50819909  devote: 1585046  devilish: 290565  ]
Autocomplete for 'mat': [ material: 98528318  math: 28631905  mateo: 2277443  ]
Autocomplete for 'esd': [ esds: 155538  ]
Autocomplete for 'hea': [ heather: 9170875  headphone: 2686393  healthier: 2463491  ]
